common/djangoapps/student/models.py

```diff
@@ -2735,6 +2735,10 @@ def activate(self):
 class AccountRecovery(models.Model):
     """
     Model for storing information for user's account recovery in case of access loss.
+
+    .. pii:: the field named secondary_email contains pii
+    .. pii_types:: email
+    .. pii_retirement:: TODO: not sure what this means
```

Comment:
Since you have the retirement call in this PR and it's hooked in via the REST API that is local to this application you can make pii_retirement as local_api.

Reply:
thanks!